lib/diagnostic.js

```diff
@@ -38,19 +46,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: "Mr.Paws",
+  species: "dog",
+  breed: "Siberian Husky",
+  vocalize: function(vocalization){
+    let vocalization = "bark bark"
+    return vocalization;
+  }
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function() {
-  /* Your answer here */
+const Pet = function(name, species, breed, vocalization) {
+  this.name = name,
+  this.species = species,
+  this.breed = breed,
+  this.vocalization = vocalization,
+  this.vocalize = vocalize
```

Comment:
`this.vocalize = vocalize' will not work because you have not vocalize is not passed in as an argument